index.bs

```diff
@@ -384,6 +384,56 @@ scheme, is encoded in the <code>URL</code> field.
   };
 </pre>
 
+<div class="example">
+In this example we build a page supporting a device. To start with when the page
```

Comment:
This example doesn't build the whole page. Maybe "In this example, we retrieve some USB devices to add to a UI."